### ISSUE TITLE
Cant set minDate/maxDate

### ISSUE THREAD

**Issue by derfloDev** [xdan/datetimepicker#577]
I dont know if this is a bug, but line 1172 in jquery.datetimepicker.js 
`...
(if` (options.minDate && /^[\+\-](.*)$/.test(options.minDate)))
...`
prevents me from setting this option

my code:
 `this.setOptions({
    maxDate:jQuery('#date_timepicker_end').val()?jQuery('#date_timepicker_end').val():false
   })
`
where maxDate for example is "12.05.2017 09:15"

**Comment by mb13** [comment]
Hello,

I don't know if we are talking about the same issue, but I'm facing with the following problem:

In commit f8a2c0810e8de1ffc805de311aaecc94e7638d89 I was able to set "minTime". In the latest version this is not working...

**Comment by sachin** [comment]
Hi derfloDev,

I faced the same issue and find out that, latest javascript code accepts the minTime as complete date time not only time.

So specify the minTime option like
```
minTime:'2017/05/18 11:00'.
```
It will works

I was able to debug this issue by put some console line in datatimepicker.full.js around line number 2342
```
console.log(now);
console.log(options.minTime);
console.log(_xdsoft_datetime.strtotime(options.minTime));
console.log(_xdsoft_datetime.strtotime(options.minTime).getTime());
console.log(now.getTime());

if ((options.minDateTime !== false && options.minDateTime > optionDateTime) || (options.maxTime !== false && _xdsoft_datetime.strtotime(options.maxTime).getTime() < now.getTime()) || (options.minTime !== false && _xdsoft_datetime.strtotime(options.minTime).getTime() > now.getTime())) {
								classes.push('xdsoft_disabled');
```

**Comment by mb13** [comment]
@sachin

unfortunately this

`minTime:'2017/05/18 15:00'`

is not working for me. I can select time after the current time and not after the time what is specified.

**Comment by derfloDev** [comment]
@mb13 

Yes, its the same problem as mine. Exept i'm using european time format.
This: `minTime: "12.05.2017 09:15"` wont work.

**Comment by mb13** [comment]
Could someone fix this issue?

The bug was put in this change:

`git diff f8a2c0810e8de1ffc805de311aaecc94e7638d89..81bca307a9777704234fec45627c09bf4d052fb0`

and since then it is not working (end of 2015).

Thanks in advance

**Comment by manhquang144** [comment]
This works for me:
minTime: new Date(new Date().setHours(8)),
